builder/builder.go

```diff
@@ -939,3 +939,128 @@ func createCompressedArchive(dst string, srcs ...string) (err error) {
 	}
 	return ioutil.WriteFile(dst, archive.Bytes(), 0644)
 }
+
+// BuildDeltaPacks between two versions of the mix.
+func (b *Builder) BuildDeltaPacks(from, to uint32) error {
+	var err error
+
+	// TODO: Configuration parsing should handle this validation/conversion.
+	var mixVersion uint32
+	mixVersion, err = parseUint32(b.Mixver)
+	if err != nil {
+		return errors.Wrapf(err, "couldn't parse mix version to use as target")
+	}
+
+	if to == 0 {
+		to = mixVersion
+	} else {
+		if to > mixVersion {
+			return errors.Errorf("--to version must be at most the latest mix version (%d)", mixVersion)
+		}
+	}
+	if from >= to {
+		return errors.Errorf("the --from version must not be larger than the --to version")
```

Comment:
"must not be larger than **or equal to** the --to version"

Reply:
It isn't clear what gets negated by  "not": "not (larger than or equal)" or "(not larger) or equal" - isn't language wonderful? ;-}

I think this is simpler: "the --from version must be smaller than the --to version".

Reply:
Yes, the "smaller" reads better. Fixed.